### PR TITLE
Add documentation for energy meters and mark unused sensors as internal.

### PR DESCRIPTION
**Pull request by Landrash**
- Adds energy sensors to the documentation.
- Change all id variables to be lower case.
- Add 'internal: true' and 'internal: false' to sensors that are either hidden or used both internally and externally.

### CODE REVIEW

**Review by maccan-tech — DISMISSED** [review]
Looking good

File: elmatare.yaml [hunk 1]
```diff
@@ -96,69 +97,101 @@ sensor:
...
     accuracy_decimals: 3
     state_class: "total_increasing"
     device_class: "energy"
+    internal: true
   - id: "Cumulative_Reactive_Import"
     unit_of_measurement: kvarh
     accuracy_decimals: 3
-  - id: "cumulative_reactive_export"
+    internal: true
+- name: "Cumulative Reactive Export"
```
Comment (maccan-tech) [inline elmatare.yaml]: wrong indentation

File: elmatare.yaml [hunk 2]
```diff
@@ -96,69 +97,101 @@ sensor:
...
     device_class: "energy"
-  - id: "cumulative_active_export"
+  - name: "Cumulative Active Import" 
+    id: "cumulative_active_export"
     unit_of_measurement: kWh
     accuracy_decimals: 3
     state_class: "total_increasing"
     device_class: "energy"
+    internal: true
   - id: "Cumulative_Reactive_Import"
```
Comment (maccan-tech) [inline elmatare.yaml]: Missing name

**Review by maccan-tech — CHANGES_REQUESTED** [review]
Need changes

File: elmatare.yaml (hunk 2 above)
Comment (Landrash) [reply elmatare.yaml]: Fixed in PR.

File: elmatare.yaml (hunk 1 above)
Comment (Landrash) [reply elmatare.yaml]: Fixed in PR :)

**Review by maccan-tech — APPROVED** [review]

**Pull request merged**